modules/storages/app/views/storages/admin/storages/update.turbo_stream.erb

```diff
@@ -0,0 +1,6 @@
+<%= turbo_stream.update ActionView::RecordIdentifier.dom_id(@storage, :storage_view_general_info) do %>
+  <%= render(::Storages::Admin::StorageViewComponent.new(@storage)) %>
+<% end %>
+<%= turbo_stream.update ActionView::RecordIdentifier.dom_id(@storage, :edit_storage_header) do %>
```

Comment:
💡 we can do without the absolute namespacing as dom_id is available as a view helper method 😉 

```suggestion
<%= turbo_stream.update dom_id(@storage, :storage_view_general_info) do %>
  <%= render(::Storages::Admin::StorageViewComponent.new(@storage)) %>
<% end %>
<%= turbo_stream.update dom_id(@storage, :edit_storage_header) do %>
```